shared-module/src/locales/en/main-frontend.json

```diff
@@ -328,6 +328,7 @@
   "other-language": "other",
   "override-completion-registration-link": "Override completion registration link",
   "page-deletion-confirmation-message": "Are you sure you want to delete page {{title}}?",
+  "page-number": "PAGE {{page-number}}",
```

Comment:
Write with normal case, and use `text-transform: uppercase` in the component where this is used.